cadquery/occ_impl/shapes.py

```diff
@@ -1914,7 +1925,8 @@ def positionAt(
     def positions(
         self: Mixin1DProtocol, ds: Iterable[float], mode: ParamMode = "length",
     ) -> List[Vector]:
-        """Generate positions along the underlying curve
+        """
+        Generate positions along the underlying curve
```

Comment:
Perhaps add punctuation to the end of this description sentence.  Compare with positionAt.  I know it's  not consistent everywhere.